packages/flutter_tools/lib/src/aot.dart

```diff
@@ -106,6 +107,20 @@ class AotBuilder {
       throwToolExit('The aot build failed.');
     }
 
+    if (reportTimings) {
+      final PerformanceMeasurement kernel = result.performance['kernel_snapshot'];
+      PerformanceMeasurement aot;
+      if (expectSo) {
+        aot = result.performance.values.firstWhere(
+          (PerformanceMeasurement measurement) => measurement.analyicsName == 'android_aot');
+      } else {
+        aot = result.performance.values.firstWhere(
+          (PerformanceMeasurement measurement) => measurement.analyicsName == 'ios_aot');
+      }
+      globals.printStatus('frontend(CompileTime): ${kernel.elapsedMilliseconds} ms.');
```

Comment:
Consider adding a test that this output appears in the logs. Consider adding a note here to consult with the Dart team before removing or modifying.

Reply:
Done